internal/trie/node/header.go

```diff
@@ -11,9 +11,9 @@ import (
 func (b *Branch) encodeHeader(writer io.Writer) (err error) {
 	var header byte
 	if b.Value == nil {
-		header = 2 << 6
+		header = byte(BranchType) << 6
```

Comment:
```suggestion
                const nodeHeaderShift = 6
		header = byte(BranchType) << nodeHeaderShift
```

Reply:
Nice name :+1: Added here and in 2 other places in that file.